src/domain/mainwindow.py

```diff
@@ -401,23 +404,34 @@ def save_config_to_file(self):
         """Method to save configuration to file."""
 
         logger.info("Saving configuration to file...")
-        # Serializing cameras per class type
+        # Prepare serialization for cameras per class type
         cameras_to_dict = []
         for camera in cameras:
             if (
                 camera.type == Camera.CameraTypes.USBCamera
                 or camera.type == Camera.CameraTypes.FTPCamera
             ):
                 cameras_to_dict.append(camera.serialize())
-        # Serialize configured notification methods
+        # Prepare serialization for notifiers per class type
         notification = {}
         for key in Notifier.notifiers:
-            notification[key] = Notifier.notifiers[key].serialize()
+            if key and Notifier.notifiers[key]:
+                notification[key] = Notifier.notifiers[key].serialize()
```

Comment:
for key, value in Notifier.notifiers.items():
            if key and value:
                notification[key] = value.serialize()